main/main_timer_sync.cpp

```diff
@@ -30,6 +30,11 @@
 
 #include "main_timer_sync.h"
 
+#ifdef DEBUG_ENABLED
+// enable to get diagnostics from here
+// #define SYNC_TIMER_DEBUG_ENABLED
+#endif
```

Comment:
This does nothing.

Reply:
> This does nothing.

Did you mean having the compile time diagnostics? While in many cases I would agree, as someone else who works on the timing code, I personally would find this more useful than to exclude it.

Reply:
What I mean is that, as-is, this code does nothing. Perhaps line 35 should be uncommented.

Reply:
Perhaps. But as explained in the slightly [wrong place further down](https://github.com/godotengine/godot/pull/35617#discussion_r371824901), `SYNC_TIMER_DEBUG_ENABLED` just guards debug output statements that I'd rate as noise for anyone not explicitly dealing with problems in this area. Some of them do trigger often at program start or when breakpoints are set/hit in the editor.

Two alternative suggestions:

1. Change the comment to `// uncomment the following line to enable extended diagnostics from this source file` to make the purpose clearer
2. Remove the block and all the logging guarded by `SYNC_TIMER_DEBUG_ENABLED`. It's not that useful and yeah, code commented out and left around "in case we need it later" is not pretty